src/providers/bttv/BttvLiveUpdates.hpp

```diff
@@ -44,15 +47,11 @@ class BttvLiveUpdates : public BasicPubSubManager<BttvLiveUpdateSubscription>
      */
     void partChannel(const QString &id);
 
-protected:
-    void onMessage(
-        websocketpp::connection_hdl hdl,
-        BasicPubSubManager<BttvLiveUpdateSubscription>::WebsocketMessagePtr msg)
-        override;
+    void stop();
```

Comment:
This stop function, just like the 7TV one, also looks unused. looks to also break tests so remove with caution